web-app/css/custom.css

```diff
@@ -8556,4 +8556,11 @@ li.survey-question.answered .survey-question-answer .option-extra-content textar
 
 .displayNone {
     display:none;
+}
+
+
+/* Closed Campaign error view */
+.closedCampaignErrorTitle {
+    font-weight: bolder;
+    color: darkred;
```

Comment:
Don't use the predefined colors. They change between browsers. Use #XXXXX. 
There is a predefined red color. Please use it.

Reply:
where can I find the predifined colors?